src/test/java/org/openmrs/performance/http/DoctorHttpService.java

```diff
@@ -284,46 +284,67 @@ public HttpRequestActionBuilder saveCondition(String patientUuid, String current
+			order.put("careSetting", CARE_SETTING_UUID);
+			order.put("concept", ASPRIN_CONCEPT_UUID);
+			order.put("dose", 1);
+			order.put("doseUnits", TABLET);
+			order.put("dosingInstructions", "");
+			order.put("dosingType", DEFAULT_DOSING_TYPE);
+			order.put("drug", ASPRIN_DRUG_UUID);
+			order.put("duration", null);
+			order.put("durationUnits", DAYS);
+			order.put("encounter", null);
+			order.put("frequency", ONCE_DAILY);
+			order.put("numRefills", 0);
+			order.put("orderReasonNonCoded", "reason");
+			order.put("orderer", session.getString("currentUserUuid"));
+			order.put("patient", session.getString("patient_uuid"));
+			order.put("quantity", 1);
+			order.put("quantityUnits", TABLET);
+			order.put("route", ORAL);
+			order.put("type", "drugorder");
+
+			Map<String, Object> encounter = new HashMap<>();
+
+			encounter.put("encounterType", ORDER);
+			encounter.put("location", OUTPATIENT_CLINIC_LOCATION_UUID);
+			encounter.put("patient", session.getString("patient_uuid"));
+			encounter.put("visit", session.getString("visitUuid"));
+			encounter.put("obs", new Object[0]);
+			encounter.put("orders", new Object[] { order });
+			encounter.put("encounterDatetime", CommonUtils.getCurrentDateTimeAsString());
 			try {
-				Map<String, Object> order = new HashMap<>();
-				order.put("action", "NEW");
-				order.put("asNeeded", false);
-				order.put("asNeededCondition", null);
-				order.put("careSetting", CARE_SETTING_UUID);
-				order.put("concept", ASPRIN_CONCEPT_UUID);
-				order.put("dose", 1);
-				order.put("doseUnits", TABLET);
-				order.put("dosingInstructions", "");
-				order.put("dosingType", DEFAULT_DOSING_TYPE);
-				order.put("drug", ASPRIN_DRUG_UUID);
-				order.put("duration", null);
-				order.put("durationUnits", DAYS);
-				order.put("encounter", session.getString("visitUuid"));
-				order.put("frequency", ONCE_DAILY);
-				order.put("numRefills", 0);
-				order.put("orderReasonNonCoded", "reason");
-				order.put("orderer", session.getString("currentUserUuid"));
-				order.put("patient", session.getString("patient_uuid"));
-				order.put("quantity", 1);
-				order.put("quantityUnits", TABLET);
-				order.put("route", ORAL);
-				order.put("type", "drugorder");
-
-				Map<String, Object> encounter = new HashMap<>();
-
-				encounter.put("encounterType", ORDER);
-				encounter.put("location", OUTPATIENT_CLINIC_LOCATION_UUID);
-				encounter.put("patient", session.getString("patient_uuid"));
-				encounter.put("visit", session.getString("visitUuid"));
-				encounter.put("obs", new Object[0]);
-				encounter.put("orders", new Object[] { order });
-				encounter.put("encounterDatetime", CommonUtils.getCurrentDateTimeAsString());
-
 				return new ObjectMapper().writeValueAsString(encounter);
 			}
 			catch (JsonProcessingException e) {
 				throw new RuntimeException(e);
 			}
+		})).check(jsonPath("$.uuid").saveAs("orderUuid"));
+	}
+
+	public HttpRequestActionBuilder deleteDrugOrder() {
```

Comment:
I think this would better describe what we do inside the method:

```suggestion
	public HttpRequestActionBuilder discontinueDrugOrder() {
```